### ISSUE TITLE
.xpi file doesn’t exist

### ISSUE THREAD

**Issue by ParkerRobb** [mronkko/ZoteroQuickLook#35]
I can’t find the *zoteroquicklook.xpi* file (or the *Releases* folder) mentioned in the README.

**Comment by bwiernik** [comment]
It's now `zoteroquicklook.zoteroplugin`

**Comment by ParkerRobb** [comment]
Well geez, nobody told me I had to go to the separate “Releases” section. I was looking for a *Releases* folder in the repository ...

**Comment by ParkerRobb** [comment]
Zotero won’t install a .zoteroplugin file, but it seems to work fine if I change the extension to .xpi and then install it ...

**Comment by bwiernik** [comment]
Zotero will install it. You can drag it onto the add-ins window like like if it has an xpi extension.

**Comment by midas0441** [comment]
hmmmm.... I don't see either `zoteroquicklook.zoteroplugin` or `zoteroquicklook.xpi`. I'm missing something obvious??

**Comment by bwiernik** [comment]
https://github.com/mronkko/ZoteroQuickLook/releases/latest

**Comment by midas0441** [comment]
thank you!

**Comment by WarthogARJ** [comment]
I had the same problem. Can you please update the Installation instructions?
I only saw how to do it by looking in Issues.

**Comment by long6-6-6** [comment]
I also had that problem. This cna't be installed because it is incompatible for the zotero6.0.5.

**Comment by Chena2003** [comment]
> I also had that problem. This cna't be installed because it is incompatible for the zotero6.0.5.

yep, I also had that problem too😢, Do you fix this and install successfullly

**Comment by long6-6-6** [comment]
This .xpi file could be installed https://github.com/l0o0/ZoteroPlugins/tree/main/docs, and you should choose the Version number named 1.4.3.
